MidasFrontend/feArduino.cxx

```diff
@@ -149,129 +138,37 @@ HNDLE handleMove;
 
 void start_move(INT hDB, INT hkey, void *info)
 {
-  // TOFIX: add some checks that we aren't already moving
-
   if(!gStartMove) return; // Just return if move not requested...
 
-  // Little magic to reset the key to 'n' without retriggering hotlink
-  BOOL move = false;
-  db_set_data_index1(hDB, handleMove, &move, sizeof(move), 0, TID_BOOL, FALSE);
+  printf("================================================================================\n");
+  printf("START MOVE\n");
+  printf("--------------------------------------------------------------------------------\n");
 
-  string path;
-  path += "/Equipment/";
-  path += EQ_NAME;
-  path += "/Settings";
+  INT status;
 
   // Get the destination position (absolute distance)
-  string destpath = path + "/Destination";
   float destination[2] = {0,0};
   int size_dest = sizeof(destination);
-  if (db_get_value(hDB, 0, destpath.c_str(), &destination, &size_dest, TID_FLOAT, TRUE) != DB_SUCCESS) {
-      // TODO error message
+  if ((status = db_get_value(hDB, 0, ODB_VAR_DESTINATION, &destination, &size_dest, TID_FLOAT, TRUE)) != DB_SUCCESS) {
```

Comment:
finding kinda hard to read, maybe would be nicer if it was split on function calls that get assigned to variables with and then check condition based on those variables?

Reply:
Sure, this is a fairly common paradigm that's kind of up for debate as to whether or not it's bad practice. I don't feel too strongly about it so I can change it.